### DISCUSSION TITLE
유니티에서 zepetopakage 파일을 만들 수 없습니다.

### DISCUSSION THREAD

**Discussion by KMG5779** [naverz/zepeto-studio-kor#616] [Issue Report (Legacy)]
publish 버튼을 누르면 진행 되다 System.TypeLoadException 오류가 뜨면서 실패하는데 원인이 뭘까요? 월드는 빌드잇에서 작업하던 월드를 불러온거고 나머진 멀티플레이 룸 가이드 예시 그대로 했습니다만...

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 보다 정확한 원인을 파악하기 위해서는 개발 환경 및 오류 스크린샷을 첨부해주시면 도움이 될 것 같습니다.
https://github.com/naverz/zepeto-studio-kor/discussions/546

**Reply by KMG5779** [reply]
개발환경은 공식 가이드 라인의 멀티플레이 룸 프로젝트에 빌드잇에서 만든 월드 맵을 맵 로더로 불러온 상태에서 퍼블리쉬를 시도한 상태입니다. 
![에러1](https://user-images.githubusercontent.com/22611586/163291286-bd9b8c27-a248-4fe1-a652-7b3f9488d518.PNG)

**Reply by KMG5779** [reply]
제페토 패키지 버전 정보는 이하와 같습니다.
![버전](https://user-images.githubusercontent.com/22611586/163291385-2200fcda-685e-4d85-a1e8-b66de68f1491.PNG)

**Reply by KMG5779** [reply]
스크립트는 일절 건드리지 않았고 Play On ZEPETO에서는 전혀 문제없이 멀티플레이가 되는 것까지 확인한 상태입니다.

**Reply by KMG5779** [reply]
개발환경은 윈도우10, 제페토 앱 버전은 3.11.3입니다

**Reply by Zepeto-TechEvangelist** [reply]
넵 올려주신 내용을 바탕으로 확인해 보겠습니다.

**Reply by Zepeto-TechEvangelist** [reply]
월드 버전을 1.1.7로 업데이트 하시고 다시 pulish 해 보시겠어요?

**Reply by KMG5779** [reply]
해결됐습니다 그런데 테스트 접속 링크의 사용법 도 좀 알수 있을까요?

**Reply by Zepeto-TechEvangelist** [reply]
넵, 연관 문제를 1.1.7에서 수정해서 해결되었다니 다행입니다.😄
테스트 접속 링크 관련해서는 아래 가이드를 참고 부탁 드립니다.
https://studio.zepeto.me/guides/testing-your-world
